README.md

```diff
@@ -1 +1,44 @@
-# calisthenics-guide
\ No newline at end of file
+# Calisthenics Expo
+
+>  This project is a page for a no existing calisthenics expo. The design authorship is from Cindy Shin.
+
+## Desktop 💻
+![screenshot](images/ssDesktop.png)
+
+
+## Mobile 📱
+![screenshot](images/ssMobile.png)
+
```

Comment:
- [ ] Hello. Don't forget to add a live demo link to your project in your readme file.